stdlib/2/commands.pyi

```diff
@@ -3,3 +3,6 @@ from typing import Tuple
 def getstatus(file: str) -> str: ...
 def getoutput(cmd: str) -> str: ...
 def getstatusoutput(cmd: str) -> Tuple[int, str]: ...
+
+def mk2arg(head: str, x: str) -> str: ...
```

Comment:
Strictly speaking this should have a similar signature to `os.path.join` (https://github.com/python/typeshed/blob/master/stdlib/2/os/path.pyi#L64), but I think it's simpler and almost correct to make all the arguments and return values be Text.

Reply:
Thank you, I'll look at this in more detail

Reply:
I made a small adjustment to the overloads you added—merging the last two into one still works correctly. Text (=unicode) in Python 2 automatically includes bytes (=str).